docs/standard/technical/deployment.md

```diff
@@ -50,7 +50,7 @@ bundle exec rake release:review_extensions
     You can skip this step if you are not releasing a new major, minor or patch version.
 ```
 
-Each release of the standard should pin to specific versions of each [core extension](http://standard.open-contracting.org/latest/en/extensions/#core-extensions). Community extensions are not pinned.
+[Core extensions](http://standard.open-contracting.org/latest/en/extensions/#core-extensions) are versioned with the standard; for example, for OCDS version 1.1.3, core extensions have releases `v1.1.3`. Core extensions have no releases that do not correspond with an OCDS version. If a core extension has not changed between patch versions of OCDS, we might not create a new release.
```

Comment:
> Core extensions have no releases that do not correspond with an OCDS version.

I'm not 100% certain this is necessarily implied by the idea that we version OCDS releases with core - and this would warrant more discussion.

My understanding is that we are aiming to  support the user story:

"When I review the 1.X documentation, the versions of core extensions I am looking at should have been reviewed for compatibility with OCDS 1.x, and the content they contain agreed as recommended for consideration for anyone publishing 1.x data".

It is quite possible that, between two versions of the standard, the master branch of an extension will evolve, even to the point where we want to make a release of it. E.g. 'bids' becomes more sophisticated as an extension containing more information in response to user demand. 

When we come to work on OCDS 1.x+1, it is possible that:

* The governance process will *reject* the idea that these additions to bids should be recommended as part of core, in which case we might end up with OCDS 1.x+1 remaining pinned to the *earlier* version of the extension;
* The governance process will highlight some other tweaks needed to the most recent release of the extension, leading to a new release of it (which might no longer be in sync with OCDS version numbers)

OK. Taking all the above into account - I think all of the change is not correct. We need to be able to pin any version of an extension to a standard release, not be tied to versioning them in lockstep.

Reply:
The paragraph is descriptive of existing practice (rather than prescriptive), so as a description, it seems correct to me.

The core extensions are [normative](https://docs.google.com/document/d/1xjlAneqgewZvHh6_hwuQ98hbjxRcA2IUqOTJiNGcOf8/edit), and therefore there is no situation in which a release is made of a core extension without that release having been reviewed through the governance process. Changes can be made on master in between OCDS versions (the master branch of an extension functions like the 1.1-dev branch of OCDS), but we don't (or shouldn't) encourage use of master branches of core extensions.

It's certainly possible that we will choose to publish releases of core extensions at a faster rate than releases of OCDS. However, this is not yet the case, and as the handbook is primarily a descriptive resource, I don't think it's appropriate to put speculative content there.

That said, it is easy to change the code and docs that presently assume lockstep versioning, so I'll update those, and then update this paragraph to describe the non-lockstep situation.

Reply:
> "The paragraph is descriptive of existing practice (rather than prescriptive), so as a description, it seems correct to me."

There can be more than one way of reading existing practice. 

> The core extensions are normative, and therefore there is no situation in which a release is made of a core extension without that release having been reviewed through the governance process. 

I believe that the intent was that the pinned versions of the core extensions, incorporated into the documentation of a given version of the standard, are normative for that given version of the standard. 

> However, this is not yet the case, and as the handbook is primarily a descriptive resource, I don't think it's appropriate to put speculative content there.

It is important, however, not to make changes that preclude things before (a) they have been discussed; and (b) on the basis only of one iteration of practice to date.